test/gpu/jit.cpp

```diff
@@ -56,9 +59,11 @@ int main() {}
 // NOLINTNEXTLINE
 const std::string add_2s_binary = R"__migraphx__(
 #include <hip/hip_runtime.h>
+#include <migraphx/kernels/types.hpp>
+using namespace migraphx;
 
 extern "C" {
-__global__ void add_2(std::int8_t* x, std::int8_t* y) 
+__global__ void add_2(int8_t* x, int8_t* y) 
```

Comment:
Just used `char` and skip including migraphx header.

Reply:
Done.